KubeArmor/core/kubeUpdate.go

```diff
@@ -1629,6 +1633,21 @@ func (dm *KubeArmorDaemon) WatchHostSecurityPolicies() {
 // == HostPolicy Backup & Restore == //
 // ================================= //
 
+// removeBackUpPolicy Function
+func (dm *KubeArmorDaemon) removeBackUpPolicy(name string) {
+
+	fname := cfg.PolicyDir + name + ".yaml"
+	// Check for "/opt/kubearmor/policies" path. If dir not found, create the same
+	if _, err := os.Stat(fname); err != nil {
+		kg.Warnf("Backup policy [%v] not exist", fname)
```

Comment:
This shoud not be a Warnf since if there is no policy applied before then this Warning will always be printed. It is better to have this as info.